code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -2,28 +2,27 @@
 
 /mob/living/carbon/human/handle_shock()
 	..()
-	if(status_flags & GODMODE || analgesic || (species && species.flags & NO_PAIN)) return //Godmode or some other pain reducers. //Analgesic avoids all traumatic shock temporarily
+	if(status_flags & GODMODE || analgesic || (species && species.flags & NO_PAIN) || stat == DEAD || !traumatic_shock)
+		return //Godmode or some other pain reducers. //Analgesic avoids all traumatic shock temporarily
 
-	switch(CLAMP(traumatic_shock, 0, 10000))
-		if(10000 to 200)
-			shock_stage = CLAMP(shock_stage++,150,200) //Indescribable pain. At this point, you will immediately be knocked down, with shock stage set to 150.
-			return
+	switch(CLAMP(traumatic_shock,0,400))
+		if(201 to 400)
```

Comment:
You would get the same behavior by clamping it to 201 and then running an `if(201)`

Reply:
I'm aware, I clamped it to the likely highest attainable level during testing; this works fine though.